usefulUtilities/ezStatusUpdate/ezStatusUpdate_Client.js

```diff
@@ -0,0 +1,70 @@
+//
+//  Easy Status Updater
+//  statusUpdate.js
+//  Created by Milad Nazeri on 2019-04-02
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+//  Update Your Status
+//
+
+(function(){
+    
+    var buttonID = null;
+    var textID = "";
+    var newStatus = "";
+    var textName = "STATUS_UPDATE_TEXT";
+    var SEARCH_RADIUS = 2;
+    var username = "";
+
+    function StatusUpdater(){
+
+    }
+
+
+    // Get the Button position and text entity ID
+    function preload(id){
+        buttonID = id;
+        var buttonPosition = Entities.getEntityProperties(buttonID, "position").position;
+        textID = Entities.findEntitiesByName(textName, buttonPosition, SEARCH_RADIUS)[0];
```

Comment:
this is going to find others' statuses sometimes. one way to solve this would be to have the user put the text entity id in the userdata of the button.